src/Sylius/Behat/Page/Admin/Channel/CreatePage.php

```diff
@@ -106,6 +114,7 @@ protected function getDefinedElements(): array
             'default_locale' => '#sylius_channel_defaultLocale',
             'enabled' => '#sylius_channel_enabled',
             'locales' => '#sylius_channel_locales',
+            'menu_taxon' => '#sylius_channel_menuTaxon',
```

Comment:
I'm wondering if we shouldn't slowly introduce data attributes here as well

Reply:
We could, but I wanted to avoid the cognitive dissonance, as we right have no test data attributes in the admin panel at all.